src/Nethermind/Nethermind.Taiko/Rpc/TaikoEngineRpcModule.cs

```diff
@@ -54,7 +55,8 @@ public class TaikoEngineRpcModule(IAsyncHandler<byte[], ExecutionPayload?> getPa
         IBlockFinder blockFinder,
         IShareableTxProcessorSource txProcessorSource,
         IRlpStreamDecoder<Transaction> txDecoder,
-        IL1OriginStore l1OriginStore) :
+        IL1OriginStore l1OriginStore,
+        ISurgeConfig surgeConfig) :
```

Comment:
Is it always available, even if taiko only runs?

Reply:
I believe so, @dipkakwani  to confirm